Person.js

```diff
@@ -0,0 +1,14 @@
+
+var Person = function(first, last, middle) {
+    this.first = first;
+    this.middle = middle;
+    this.last = last;
+};
+
+Person.prototype = {
+
+    whoAreYou : function() {
+        return this.first + (this.middle ? ' ' + this.middle: '') + ' ' + this.last;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/srichaturya-hari-by/Test/pullRequest?prid=7220265)
```suggestion
        return this.first + (this.middle ? " " + this.middle: '') + ' ' + this.last;
```